libs/payments/paypal/src/index.ts

```diff
@@ -2,7 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-export { PayPalClient } from './lib/paypal.client';
+export * from './lib/paypal.client';
```

Comment:
Just curious, why change this one, but leave `paypal.error` and `lib/util` as is?